client/src/Apps/Authorized/UserApp.tsx

```diff
@@ -0,0 +1,23 @@
+import { FunctionComponent } from 'react'
+import { Routes, Route } from 'react-router-dom'
+import { Meeting } from '../../Pages/Meeting/Meeting'
+import { Review } from '../../Pages/Review/Review'
+import { SignUp } from '../../Pages/SignUp/SignUp'
+import { Admin } from '../../Pages/Admin/Admin'
+import { FourOFour } from '../../Pages/404/404'
+
+const UnAuthrizedApp: FunctionComponent<any> = () => {
+  return (
+    <>
+      <Routes>
+        <Route path="/signup" element={<SignUp />} />
+        <Route path="/meeting" element={<Meeting />} />
+        <Route path="/createReview" element={<Review />} />
+        <Route path="/admin" element={<Admin />} />
+        <Route path="/" element={<FourOFour />} />
+      </Routes>
+    </>
```

Comment:
맨 바깥에 있는 fragment는 빼도 될 것 같아요.
```suggestion
      <Routes>
        <Route path="/signup" element={<SignUp />} />
        <Route path="/meeting" element={<Meeting />} />
        <Route path="/createReview" element={<Review />} />
        <Route path="/admin" element={<Admin />} />
        <Route path="/" element={<FourOFour />} />
      </Routes>
```